chingu-voyages/v19-bears-team-07
# Implement a comments display section for a single app.

This pull request implements a basic display of paginated comments when viewing single apps. Right now, the comment data is just dummy data that'll have to be replaced once the backend for comments is implemented. I also only display 3 comments max on a page, at this time.

To test:

- run `rails server`
- run `gatsby develop`
- visit `/apps`
- Click on an app, and scroll to the bottom of the screen to see the comments.
- Click on a page number to view a different page of the comments.

![comments](https://user-images.githubusercontent.com/62522740/82261635-3ec95900-9914-11ea-96fb-7f2359163943.PNG)
